io.github.bruttazz.FirefoxDevFlatpak.yml

```diff
@@ -0,0 +1,92 @@
+      # neither the fiefox archive supports 'latest' tag (https://archive.mozilla.org/pub/devedition/releases)
+      # raise a pr if you managed to find a better way 
+      - wget -O firefox.tar.bz2 'https://download.mozilla.org/download/en-US/?product=firefox-devedition-latest-ssl&os=linux64&lang=en-US'
+      - mkdir -p "$FLATPAK_DEST/opt"
+      - mv firefox.tar.bz2 "$FLATPAK_DEST/opt/firefox.tar.bz2"
+      - tar xvf "$FLATPAK_DEST/opt/firefox.tar.bz2" -C "$FLATPAK_DEST/opt/"
+      - mkdir -p "$FLATPAK_DEST/bin"
+      - ln -s "$FLATPAK_DEST/opt/firefox/firefox" "$FLATPAK_DEST/bin/firefox"
+
+  - name: integrations
+    buildsystem: simple
+    
+    sources:
+      - type: file
+        path: data/io.github.bruttazz.FirefoxDevFlatpak.desktop
+        dest-filename: io.github.bruttazz.FirefoxDevFlatpak.desktop
+
+      - type: file
+        path: data/io.github.bruttazz.FirefoxDevFlatpak.metainfo.xml
+        dest-filename: io.github.bruttazz.FirefoxDevFlatpak.metainfo.xml
+
+    build-commands:
+      - mkdir -p "$FLATPAK_DEST/share/applications/"
+      - >
+        mv io.github.bruttazz.FirefoxDevFlatpak.desktop 
+        "$FLATPAK_DEST/share/applications/io.github.bruttazz.FirefoxDevFlatpak.desktop"
+
+      - mkdir -p "$FLATPAK_DEST/share/appdata/"
+      - >
+        mv io.github.bruttazz.FirefoxDevFlatpak.metainfo.xml
+        "$FLATPAK_DEST/share/appdata/io.github.bruttazz.FirefoxDevFlatpak.metainfo.xml"
+
+      # ABOUT THE ICON COPYING : If the base package changed their convention, then there is a catch
+      - mkdir -p "$FLATPAK_DEST/share/icons/hicolor/128x128/apps/"
+      - >
+        cp "$FLATPAK_DEST/opt/firefox/browser/chrome/icons/default/default128.png" 
+        "$FLATPAK_DEST/share/icons/hicolor/128x128/apps/io.github.bruttazz.FirefoxDevFlatpak.png"
+
+      - mkdir -p "$FLATPAK_DEST/share/icons/hicolor/64x64/apps/"
+      - >
+        cp "$FLATPAK_DEST/opt/firefox/browser/chrome/icons/default/default64.png"
+        "$FLATPAK_DEST/share/icons/hicolor/64x64/apps/io.github.bruttazz.FirefoxDevFlatpak.png"
+
+      - mkdir -p "$FLATPAK_DEST/share/icons/hicolor/32x32/apps/"
+      - >
+        cp "$FLATPAK_DEST/opt/firefox/browser/chrome/icons/default/default32.png"
+        "$FLATPAK_DEST/share/icons/hicolor/32x32/apps/io.github.bruttazz.FirefoxDevFlatpak.png"
+
+      - mkdir -p "$FLATPAK_DEST/share/icons/hicolor/16x16/apps/"
+      - >
+        cp "$FLATPAK_DEST/opt/firefox/browser/chrome/icons/default/default16.png"
+        "$FLATPAK_DEST/share/icons/hicolor/16x16/apps/io.github.bruttazz.FirefoxDevFlatpak.png"
+
+      - mkdir -p "$FLATPAK_DEST/share/icons/hicolor/scalable/apps/"
+      # yes i didn't got any SVG (what if some syste going to identify it as a good thing 🫢, that's why the follwoing line)
+      - >
+        cp "$FLATPAK_DEST/opt/firefox/browser/chrome/icons/default/default128.png"
+        "$FLATPAK_DEST/share/icons/hicolor/scalable/apps/io.github.bruttazz.FirefoxDevFlatpak.png"
```

Comment:
why are you lying claiming a PNG is scalable?

Reply:
My bad, I thought in some use cases the system will only look for icons under scalable. So..